test-suite/Network/CloudSeeder/Test/Stubs.hs

```diff
@@ -39,10 +39,10 @@ instance Monad m => MonadCLI (ArgumentsT m) where
   getOptions pSpecs = ArgumentsT $ do
     input <- ask
     let x = execParserPure defaultPrefs (parseOptions pSpecs) input
-    return $ fromJust $ getParseResult x
+    return $ unsafePerformIO $ handleParseResult x
```

Comment:
getParseResult throws away all the parse failure info, only gives back a Maybe ParserResult. As far as I can tell, these are the only two functions that take a ParserResult. I suppose I could write a custom pattern match on ParserResult, but given that this is easier and it's a stub...what's the danger of using unsafePerformIO in this particular instance?